### PR TITLE
Updating to the latest releases

### PR DESCRIPTION
**Pull request by rawkode**
Updating Helm chart and Kubernetes YAMLs to faas 0.6.6b and faas-netes 0.3.1a

### CODE REVIEW

File: faas.armhf.yml [hunk 1]
```diff
@@ -68,7 +68,7 @@ spec:
     spec:
       containers:
       - name: gateway
-        image: functions/gateway:armhf-0.6.5
+        image: functions/gateway:armhf-0.6.6b
```
Comment (alexellis) [inline faas.armhf.yml]: Not sure if the arm tag was pushed.

File: openfaas/values.yaml [hunk 2]
```diff
@@ -6,11 +6,11 @@ rbac: true
 
 # images
 images:
-  controller: functions/faas-netesd:0.3.1
+  controller: functions/faas-netesd:0.3.1a
   controllerArmhf: functions/faas-netesd-armhf
 
   gateway: functions/gateway:0.6.6-beta1
-  gatewayArmhf: functions/gateway-armhf:0.6.5
+  gatewayArmhf: functions/gateway-armhf:0.6.6b
```
Comment (alexellis) [inline openfaas/values.yaml]: Likewise, not sure if this tag is updated yet?

File: openfaas/values.yaml [hunk 3]
```diff
@@ -6,11 +6,11 @@ rbac: true
 
 # images
 images:
-  controller: functions/faas-netesd:0.3.1
+  controller: functions/faas-netesd:0.3.1a
   controllerArmhf: functions/faas-netesd-armhf
 
   gateway: functions/gateway:0.6.6-beta1
```
Comment (alexellis) [inline openfaas/values.yaml]: This can be updated to latest too

**Comment by rawkode** [comment]
@alexellis This one looks OK now. Couple of greps came back clean.

**Comment by alexellis** [comment]
Thanks @rawkode

**Pull request merged**